internal/scaffold/olm-catalog/csv_test.go

```diff
@@ -145,16 +144,8 @@ func TestUpdateVersion(t *testing.T) {
 		t.Errorf("Wanted csv name %s, got %s", wantedName, csv.ObjectMeta.Name)
 	}
 
-	var resolver *olminstall.StrategyResolver
-	strategyInterface, err := resolver.UnmarshalStrategy(csv.Spec.InstallStrategy)
-	if err != nil {
-		t.Fatal(err)
-	}
-	strategy, ok := strategyInterface.(*olminstall.StrategyDetailsDeployment)
-	if !ok {
-		t.Fatalf("Strategy of type %T was not StrategyDetailsDeployment", strategyInterface)
-	}
-	csvPodImage := strategy.DeploymentSpecs[0].Spec.Template.Spec.Containers[0].Image
+	csvDepSpecs := csv.Spec.InstallStrategy.StrategySpec.DeploymentSpecs
+	csvPodImage := csvDepSpecs[0].Spec.Template.Spec.Containers[0].Image
```

Comment:
Can this panic? While unlikely, it is theoretically possible for there to be no `Containers`, right?